### PR TITLE
EZP-30627: Unified twig helpers naming

### PR DESCRIPTION
**Pull request by pawbuj**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30627](https://jira.ez.no/browse/EZP-30627)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

### renamed functions:

* ez_path_to_locations <-- ez_path_string_to_locations
* ez_block_value_encode <-- encode_block_value
* ez_http_cache_tag_location <-- ez_http_tag_location 
* ez_field_encode <-- encode_field
* ez_field_is_empty <-- ez_is_field_empty
* ez_render_field_definition_edit <-- ez_render_fielddefinition_edit
* ez_render_field_definition_settings <-- ez_render_fielddefinition_settings
* ez_render_component <-- ezplatform_admin_ui_component ??
* ez_render_component_group <-- ezplatform_admin_ui_component_group
* ez_render_tab_group <-- ez_platform_tabs
* ez_content_field_identifier_image_asset <-- ez_image_asset_content_field_identifier
* ez_content_field_identifier_first_filed_image <-- ez_first_filled_image_field_identifier
* ez_page_builder_cross_origin_helper <-- ezplatform_page_builder_cross_origin_helper

### remooved functions: 
* ez_trans_prop

### renamed globals:

* ez_admin_ui_config <-- admin_ui_config
* ezplatform <-- ezpublish 

### renamed filters:

* ez_richtext_to_html5 <-- richtext_to_html5
* ez_richtext_to_html5_edit <-- richtext_to_html5_edit

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezimageasset.html.twig [hunk 1]
```diff
@@ -84,7 +84,7 @@
             locationId: destination_content.contentInfo.mainLocationId
         }) %}
 
-        {% set image_uri = ez_field_value(destination_content, ez_image_asset_content_field_identifier()).uri %}
+        {% set image_uri = ez_field_value(destination_content, ez_content_field_identifier_image_asset()).uri %}
```
Comment (mikadamczyk) [inline src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezimageasset.html.twig]: As I said the old name fits me better. In other names, we have the object first, then "value". If nobody shares my opinion, than I am ok with this

Comment (pawbuj) [reply src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezimageasset.html.twig]: You have also said that if we have more than one function in `ez_content_field_identifie` domain then you are ok with it. I decide to keep `ez_content_field_identifier_first_filed_image ` so we have two.

**Comment by alongosz** [comment]
Quick question: when renamed `admin_ui_config` into `ez_ui_config`, what was the reason to skip `admin` from the name? You've probably discussed this somewhere, but I missed that ;)

**Comment by pawbuj** [comment]
That was my invention. The name was too detailed.

**Pull request merged**